src/main/pages/che-7/installation-guide/proc_upgrading-che-using-operatorhub.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// upgrading-{prod-id-short}
+
+[id="upgrading-{prod-id-short}-using-operatorhub_{context}"]
+= Upgrading {prod-short} using OperatorHub
+
+This section describes how to upgrade from previous minor version using the Operator from OperatorHub in the OpenShift web console.
+
+.Prerequisites
+
+* An administrator account on an OpenShift instance.
+
+* An instance of a previous minor version of {prod-short}, installed using an Operator from OperatorHub on the same instance of OpenShift.
+
+.Procedure
+
+. Open the OpenShift web console.
+
+. Navigate to the *Operators* -> *Installed Operators* section.
+
+. Click *{prod}* in the list of installed operators.
+
+. Navigate to the *Subscription* tab and enable the following options:
+
+  * *Channel*: `latest`
+  * *Approval*: `Automatic`
+
+.Verification steps
+
+. Log in to the {prod-short} instance.
```

Comment:
I don't think you have to log in to your cluster again after finishing upgrade (unless you log out, obv)... I'll need to check but I vaguely remember you just need to refresh the page or smth.

I think one step would be enough
>See the updated {prod-ver} version number is visible at the bottom of the page.
or something like this

Reply:
As in previous steps you are in OpenShift web console, you need to navigate back to Che.

Reply:
Aha, ok, makes sense